app/controllers/reviews_controller.rb

```diff
@@ -25,4 +45,10 @@ def review_params
       user: User.find(current_user)
       )
   end
+
+  def authorize_user
+    if !user_signed_in?
+      raise ActionController::RoutingError.new("Not Found")
+    end
+  end
```

Comment:
You _may_ want to investigate the Devise helper method `authenticate_user` (although yours is doing a slightly different thing, which may be intentional design -- in which case, go you!)